src/pages/Login/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import {
+  getAuth,
+  signInWithEmailAndPassword,
+  FacebookAuthProvider,
+  GoogleAuthProvider,
+  signInWithPopup,
+} from "firebase/auth";
+import { Button, Input } from "../../ components";
+import app from "../../services/firebase";
+import {
+  Alert,
+  Eye,
+  GoogleIcon,
+  FacebookIcon,
+  Hidden,
+} from "../../assets/index";
+
+import styles from "./styles.module.scss";
+import { changeUser } from "../../services/redux/userSlice";
+
+export const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [viewPassword, setViewPassword] = useState(false);
```

Comment:
I suggested use the useReducer for this, as the function is used only to toggle passwordView
ex: const [viewPassword, togglePasswordView] = useReducer(state=> !state, false)